blockdag/process.go

```diff
@@ -253,6 +253,8 @@ func (dag *BlockDAG) processBlockNoLock(block *util.Block, flags BehaviorFlags)
 		}
 	}
 
+	dag.addBlockProcessTime()
```

Comment:
I think we want to call that only if block was processed from sync, not submitBlock. No?